src/index.tsx

```diff
@@ -1,18 +1,19 @@
-import '@buerli.io/react/build/middleware/batchedUpdates'
 import 'antd/dist/antd.less'
 import './ipc'
 import React from 'react'
-import ReactDOM from 'react-dom'
+import { createRoot } from 'react-dom/client'
 import { App } from './App'
 import { initBuerli } from './initBuerli'
 import { Global } from './styles/Global'
 
 initBuerli()
 
-ReactDOM.render(
+const container = document.getElementById('app')
```

Comment:
Id has to be `root`. Will fix it in my next commit.

Reply:
Fixed https://github.com/awv-informatik/buerligons/pull/12/commits/542f06a70b883f3cb034f8d15e97b95c65f53284